usaspending_api/agency/v2/views/federal_account_list.py

```diff
@@ -0,0 +1,103 @@
+from django.db.models import Sum, Q
+from rest_framework.request import Request
+from rest_framework.response import Response
+from typing import Any, List
+from usaspending_api.agency.v2.views.agency_base import AgencyBase, ListMixin
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.helpers.generic_helper import get_simple_pagination_metadata
+from usaspending_api.financial_activities.models import FinancialAccountsByProgramActivityObjectClass
+
+
+class FederalAccountList(ListMixin, AgencyBase):
+    """
+    Obtain the list of program activity categories for a specific agency in a
```

Comment:
Program activities?  Should this be federal accounts?

Reply:
yea probably